DscResources/MSFT_GroupResource/MSFT_GroupResource.psm1

```diff
@@ -2487,7 +2487,11 @@ function Clear-GroupMember
 <#
     .SYNOPSIS
         Adds the specified member to the specified group.
-        This is a wrapper function for testing purposes.
+
+        There is an issue reported at https://github.com/PowerShell/PSDscResources/issues/82
```

Comment:
Can you move this block into the .NOTES section?

Reply:
Updated as per request